src/components/Input/index.tsx

```diff
@@ -1,27 +1,26 @@
 import React, { InputHTMLAttributes } from 'react';
 import i18next from 'i18next';
 
-import { I18N_CONFIG } from 'constants/index';
-
 import styles from './styles.module.scss';
 
 interface Props extends InputHTMLAttributes<HTMLInputElement> {
-  inputRef?: React.Ref<HTMLInputElement>;
+  inputRef: React.Ref<HTMLInputElement>;
   label?: string;
   errorMessage?: string;
+  i18nKey: string;
 }
 
-function InputText({ inputRef, label, errorMessage, name, ...rest }: Props) {
+function InputText({ inputRef, label, errorMessage, name, i18nKey, ...rest }: Props) {
```

Comment:
This prop seems strange to me, I would prefer that you send it directly in the "name" prop